charts/podinfo/templates/deployment.yaml

```diff
@@ -87,6 +102,17 @@ spec:
             - name: http
               containerPort: {{ .Values.service.httpPort | default 9898 }}
               protocol: TCP
+              {{- if .Values.service.hostPort }}
+              hostPort: {{ .Values.service.hostPort }}
+              {{- end }}
+            {{- if .Values.tls.enabled }}
+            - name: https
+              containerPort: {{ .Values.tls.port | default 98899 }}
```

Comment:
Shouldn't this be `default  9899`?

Reply:
fixed -- good catch. I blame key ghosting. O.o